deployments-review/apps/expense-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+quarkus.swagger-ui.always-include=true
+
+#quarkus.datasource.devservices.image-name = docker.io/library/postgres:latest
+quarkus.hibernate-orm.sql-load-script=META-INF/import.sql
+quarkus.hibernate-orm.database.generation=create
+#quarkus.datasource.username = ${QUARKUS_DATABASE_USER}
+#quarkus.datasource.password = ${QUARKUS_DATABASE_PASSWORD}
+#quarkus.datasource.jdbc.url = jdbc:postgresql://postgresql:5432/${QUARKUS_DATABASE_NAME}
```

Comment:
I see a QUARKUS_xxx pattern but in the deployment resource the DATABASE_NAME env var is used... does that correspond with this configuration?